opencodeblocks/blocks/codeblock.py

```diff
@@ -112,10 +195,21 @@ def update_output_panel(self):
             self.output_closed = True
             self.splitter.setSizes([1, 0])
 
+    def update_run_all_button(self):
+        """ Change the geometry of the run all button """
+        if hasattr(self, 'run_all_button'):
+            self.run_all_button.setGeometry(
+                int(self.width - self.edge_size - self.run_all_button.width()),
+                int(self.edge_size / 2),
+                int(self.run_all_button.width()),
+                int(self.run_all_button.height()),
+            )
+
```

Comment:
Useless if button is fixed and placed.

```suggestion
```

Reply:
I have tried fixing and moving the button and the button stays at its initial position and does not stay in the upper right corner when resizing the window. I am keeping the code as is for now

Reply:
move is better ! That's perfect as is now !